### PR TITLE
Fixing an issue with concurrent file copy

### PR DESCRIPTION
**Pull request by agirish**

### CODE REVIEW

**Comment by Agirish** [comment]
Chun, please review. I have a few successful runs (both Functional and Advanced). Have a few more scheduled - will share results with you offline.

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java [hunk 1]
```diff
@@ -501,9 +502,11 @@ public void cancel() {
           @Override
           public void run() {
             try {
-              Path src = new Path(CWD + "/" + Utils.getDrillTestProperties().get("DRILL_TEST_DATA_DIR"), datasource.src);
+              Path src = new Path(CWD + "/" + Utils.getDrillTestProperties().get("DRILL_TEST_DATA_DIR") + "/" + datasource.src);
               Path dest = new Path(drillTestData, datasource.dest);
-              hdfsCopy(src, dest, false, fsMode);
+
+              LOG.debug("Copy " + src + " to " + dest);
```
Comment (cchang738) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]: I think it's better leave the debug log inside the function. If you have multiple calls to the function, you don't need to add multiple LOG lines.

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java [hunk 2]
```diff
@@ -555,34 +558,32 @@ public void run() {
...
-    FileSystem localFs = FileSystem.getLocal(conf);
-    if (localFs.getFileStatus(src).isDir()) {
-      for (FileStatus file : localFs.listStatus(src)) {
-        Path newSrc = file.getPath();
-        Path newDest = new Path(dest, newSrc.getName());
-        hdfsCopy(file.getPath(), newDest, overWrite, fsMode);
-      }
-    } else if (!fs.exists(dest) || overWrite) {
+
+    if (!fs.exists(dest)) {
```
Comment (cchang738) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]: What happens to the "overWrite" case? Have you tested force write?

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java (hunk 1 above)
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]: Moved it inside.

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java (hunk 2 above)
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]: We've never supported overWrite, it was hardcoded to false. So I just removed it. We can check support for overWrite on a later change.

File: framework/src/main/java/org/apache/drill/test/framework/TestDriver.java [hunk 3]
```diff
@@ -501,9 +502,11 @@ public void cancel() {
           @Override
           public void run() {
             try {
-              Path src = new Path(CWD + "/" + Utils.getDrillTestProperties().get("DRILL_TEST_DATA_DIR"), datasource.src);
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestDriver.java]: No I guess it was just to make it more clear.

**Comment by Agirish** [comment]
I found an issue. Closing this PR. Will reopen one I have a fix.

**Pull request closed without merging**